File: easydrop-front/src/app/models/storehouse.ts
export class Storehouse {
  idStorehouse: string;
  idSeller: string;
  storehouseName: string;
  storehouseAddress: string;
  sellerName: string;
  imgURL: string;
  xcoord: number;
  ycoord: number;

  constructor(idSeller: string, storehouseName: string, sellerName: string,
              storehouseAddress: string,  xCoord: number, yCoord: number, imgURL: string) {
    this.idSeller = idSeller;
    this.storehouseName = storehouseName;
    this.storehouseAddress = storehouseAddress;
    this.xcoord = xCoord;
    this.ycoord = yCoord;
    this.imgURL = imgURL;
    this.sellerName = sellerName;
  }

  setIdStorehouse(idStorehouse: string) {
    this.idStorehouse = idStorehouse;
  }

  printDetails() {
    alert(this.idSeller + " " + this.storehouseName + " " + this.storehouseAddress + " " +
          this.xcoord + " " + this.ycoord)
  }
}

File: easydrop-backend/src/main/java/com/easydropbackend/entities/Storehouse.java
package com.easydropbackend.entities;

import lombok.Getter;
import lombok.Setter;
import org.hibernate.annotations.GenericGenerator;

import javax.persistence.*;

@Getter
@Setter
@Entity
@Table(name = "storehouse")
public class Storehouse {
    @Id
    @GenericGenerator(name = "uuid", strategy = "org.hibernate.id.UUIDHexGenerator")
    @GeneratedValue(generator = "uuid")
    @Column(nullable = false, updatable = false, name="id_storehouse")
    private String idStorehouse;

    @Column(name="id_seller")
    private String idSeller;

    @Column(name="storehouse_name")
    private String storehouseName;

    @Column(name="x_coord")
    private String xCoord;

    @Column(name="y_coord")
    private String yCoord;

    @Column(name="img_url")
    private String imgURL;
}
